extensions/ql-vscode/src/run-remote-query.ts

```diff
@@ -34,9 +67,13 @@ export default async function runRemoteQuery(credentials: Credentials, uri?: Uri
   const config = yaml.safeLoad(await fs.readFile(repositoriesFile, 'utf8')) as Config;
 
   const ref = config.ref || 'main';
-  const language = config.language;
+  const language = config.language || await findLanguage(cliServer, uri);
   const repositories = config.repositories;
 
+  if (!language) {
+    return;
```

Comment:
I would add a brief comment stating that we don't need an error message here because `findLanguage` already displays one.